core/arch/opnd.h

```diff
@@ -225,6 +225,22 @@ enum {
     DR_REG_XMM13,
     DR_REG_XMM14,
     DR_REG_XMM15,
+    DR_REG_XMM16,
```

Comment:
These need to go at the end, not in the middle.

Reply:
New proposal (offline) does add in the middle.